.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Please describe your changes -->
+
+## How Has This Been Tested?
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+## Screenshots (jpeg or gifs if applicable):
+
+## Types of changes
+<!-- What types of changes does your code introduce?  -->
+<!-- Bug fix (non-breaking change which fixes an issue) -->
+<!-- New feature (non-breaking change which adds functionality) -->
+<!-- Breaking change (fix or feature that would cause existing functionality to not work as expected) -->
+
+## Checklist:
+- [ ] My code is tested.
+- [ ] My code follows the WordPress code style.
+- [ ] My code follows has proper inline documentation.
```

Comment:
A link to the [TESTING](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#testing) section of `CONTRIBUTING.md` should be included in this section

Reply:
I would say don't link as it could create the issue we already have. I know it's a good information source, but we consistently have an issue with people not clearing out this content.